core/db_classes/EE_Form_Section.class.php

```diff
@@ -225,7 +225,7 @@ public function setStatus(string $status)
      * @throws EE_Error
      * @throws ReflectionException
      */
-    public function wpUser(): int
+    public function wp_user(): int
```

Comment:
is this so things match up with WP GQL?

Reply:
Yes, all other model objects use `wp_user`